src/components/PartnerBonusFormSteps.tsx

```diff
@@ -41,12 +41,12 @@ export const IncomeQuestion = ({ disabled }) => {
       <Input
         name="r032_partner_vlastne_prijmy"
         type="number"
-        label="Vlastné príjmy manželky / manžela"
+        label={"Vlastné príjmy manželky / manžela za zdaňovacie obdobie " + TAX_YEAR}
```

Comment:
na taketo spajanie konstant a textu je urcite vhone pouzit [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

```suggestion
        label={`Vlastné príjmy manželky / manžela za zdaňovacie obdobie ${TAX_YEAR}`}
```